src/LiveComponent/assets/package.json

```diff
@@ -44,9 +44,6 @@
             "@symfony/ux-live-component": "path:%PACKAGE%/dist/live_controller.js"
         }
     },
-    "dependencies": {
-        "idiomorph": "^0.3.0"
-    },
```

Comment:
why ?

Reply:
See https://github.com/symfony/ux/pull/2935#discussion_r2221428147

Reply:
Yes but it is not inlined anymore, right ? cf comment on top here.... so there is something weird here.

https://github.com/symfony/ux/pull/2935#pullrequestreview-3045144241

Reply:
Nevermind i'm blind